### DISCUSSION TITLE
Reply to @Jmtatsch's question.

### DISCUSSION THREAD

**Discussion by Sam8688** [open-webui/open-webui#3772] [General]
I created a bug report, and @jmtatsch  asked me a question, but i think mistakenly, locked me out of being able to reply to the question!.

https://github.com/open-webui/open-webui/issues/3763

Here's the reply

> is url https encrypted? many browsers/os require that for giving access to the microphone.

It's not an encrypted connection.

Here's the address I use to access my OpenWebUI UI

http://100.76.182.93:30000/

I'm using Tailscale to access my server, I'm new to Linux and don't want to try and set up HTTPS certificates on my server if possible!

Thanks for the fast response.

Unrelated question.
Any plans to build an OpenWebUi Android app?
Would be really cool to have an OpenWebUi android IOS Linux Windows Mac apps in the future even if It's just web wrappers for the OpenWebUi web app

**Comment by flefevre** [comment]
Personnaly I use traefik as frontend proxy, it is configured with Https then it redirect flux with http in the same host to open WebUI.

One type of question per discussion is more easy to handle
